graphsdk/src/main/java/com/microsoft/graph/generated/BaseDirectoryObjectReferenceRequest.java

```diff
@@ -49,7 +49,7 @@ public DirectoryObject delete() throws ClientException {
      */
     public IDirectoryObjectReferenceRequest select(final String value) {
         getQueryOptions().add(new QueryOption("$select", value));
-        return (DirectoryObjectReferenceRequest)this;
+        return (DirectoryObjectReferenceRequest) this;
     }
```

Comment:
Appears to be an artifact of an existing formatting error in the template. I don't stress this stuff, but it's there.